src/terra-dev-site/contributing/Prerequisites.a.contributing.md

```diff
@@ -0,0 +1,62 @@
+
+If you decide to fix an issue, please be sure to check the comment thread in case somebody is already working on a fix. If nobody is working on it at the moment, please leave a comment stating that you intend to work on it so other people don’t accidentally duplicate your effort.
+If somebody claims an issue but doesn’t follow up for more than two weeks, it’s fine to take over it but you should still leave a comment.
+
+## Repositories
+
+Terra UI is divided into a several projects:
+* [terra-ui](https://github.com/cerner/terra-ui) - The primary documentation site for terra.
+* [terra-core](https://github.com/cerner/terra-core) -  Contains core react components to help developers quickly build out new applications.
+* [terra-framework](https://github.com/cerner/terra-framework) - Contains higher order, opinionated react components to help developers quickly build out new applications.
+* [terra-clinical](https://github.com/cerner/terra-clinical) - Contains clinical focused components to help developers quickly build out new applications.
+* [terra-toolkit](https://github.com/cerner/terra-toolkit) - A set of tools providing configuration, as well as helper functions for testing.
+* [terra-dev-site](https://github.com/cerner/terra-dev-site) - Common 'dev site' for documenting terra components.
+* [generator-terra-module](https://github.com/cerner/generator-terra-module) - Yeoman based generator for scaffolding out a new directory for terra npm packages in a monorepo.
+* [eslint-config-terra](https://github.com/cerner/eslint-config-terra) - Terra’s eslint configuration.
+* [stylelint-config-terra](https://github.com/cerner/stylelint-config-terra) - Terra’s stylelint configuration.
+* [browserslist-config-terra](https://github.com/cerner/browserslist-config-terra) - Terra’s browserslist configuration.
+
+## Development Setup
+
+### Git / GitHub
+
+You must have [git](https://help.github.com/articles/set-up-git/) installed on your computer and an account with [Github](https://github.com/).
```

Comment:
We try to adhere by these standards. Do we want to mention them?
- Use your real name in your profile
- Have a unique profile picture
- Uses 2-factor authentication

Reply:
Hmm, I left them off as contributors outside of Cerner don't need these to contribute.